### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by harmony-weblate**
Translations update from [Weblate](https://translate.harmony.tf) for [Northstar/Northstar Client Localisation](https://translate.harmony.tf/projects/northstar/client/).

Current translation status:

Weblate translation status

### CODE REVIEW

File: Northstar.Client/mod/resource/northstar_client_localisation_russian.txt [hunk 1]
```diff
@@ -62,7 +62,7 @@
 		"featured_mode_all_holopilot" "Великий обманщик"
 		"featured_mode_all_grapple" "Зацепер"
 		"featured_mode_all_phase" "Мир иной"
-		"featured_mode_all_ticks" "Горячие парни"
+		"featured_mode_all_ticks" "Крутые перцы"
```
Comment (GeckoEidechse) [inline Northstar.Client/mod/resource/northstar_client_localisation_russian.txt:65]: Checking with Google translate:

`Горячие парни` -> `Hot guys` (I guess close to `Spicy boys`?)
`Крутые перцы` -> `Cool dudes` (huh?)

So yeah, this kinda doesn't seem right...

Comment (GeckoEidechse) [reply Northstar.Client/mod/resource/northstar_client_localisation_russian.txt:65]: As per discussion on Discord, this should be `Oстрый`

Comment (WofWca) [reply Northstar.Client/mod/resource/northstar_client_localisation_russian.txt:65]: See comments on Weblate․ It's how "Spicy boy" is translated to Russian in the original Titanfall. It's "Крутой перец". Just "Острый" doesn't even sound like a game mode name.

Comment (GeckoEidechse) [reply Northstar.Client/mod/resource/northstar_client_localisation_russian.txt:65]: Aight, thanks for the clarification <3

**Review by GeckoEidechse — APPROVED** [review]
Concerns from review addressed by @WofWca

Not tested as this PR only edits translations.

**Pull request merged**